### PR TITLE
fix: Ticket query attribute in send event fee

### LINKED ISSUE #6769: Task send_event_fee_notification failing
```
raised unexpected: AttributeError("'Ticket' object has no attribute 'ticket_id'")
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 385, in trace_task
    R = retval = fun(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/flask_celeryext/app.py", line 101, in __call__
    res = Task.__call__(self, *args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/celery/app/trace.py", line 650, in __protected_call__
    return self.run(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 161, in _inner
    reraise(*exc_info)
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/_compat.py", line 57, in reraise
    raise value
  File "/usr/local/lib/python3.7/site-packages/sentry_sdk/integrations/celery.py", line 156, in _inner
    return f(*args, **kwargs)
  File "/data/app/app/api/helpers/scheduled_jobs.py", line 90, in send_event_fee_notification
    ticket = safe_query(db, Ticket, 'id', order_ticket.ticket_id, 'ticket_id')
AttributeError: 'Ticket' object has no attribute 'ticket_id'

```

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #6769

### CODE REVIEW

File: app/api/helpers/scheduled_jobs.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def send_event_fee_notification():
             fee_total = 0
             for order in orders:
                 for order_ticket in order.tickets:
-                    ticket = safe_query(db, Ticket, 'id', order_ticket.ticket_id, 'ticket_id')
+                    ticket = safe_query(db, Ticket, 'id', order_ticket.id, 'id')
```
Comment (iamareebjamal) [inline app/api/helpers/scheduled_jobs.py]: Why even do this? We already have the ticket as order_ticket?

Comment (codedsun) [reply app/api/helpers/scheduled_jobs.py]: @iamareebjamal - *safe query*.

Comment (kushthedude) [reply app/api/helpers/scheduled_jobs.py]: @iamareebjamal Done the change

Comment (iamareebjamal) [reply app/api/helpers/scheduled_jobs.py]: @codedsun If someone added a ticket in order when it was not deleted, and then placed an order and now the ticket was deleted. Should it add the fee of deleted ticket in the order invoice or not?

**Comment by codedsun** [comment]
@iamareebjamal - Will  the orders contain only ticket ids which are not deleted?

File: app/api/helpers/scheduled_jobs.py (hunk 1 above)
Comment (codedsun) [reply app/api/helpers/scheduled_jobs.py]: Yes, cool! 👍 got that

**Pull request merged**